<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aacConfigPage</class>
 <widget class="QWidget" name="aacConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AAC encoding configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="aacUseQualityCheck">
     <property name="text">
      <string>Use quality setting (recommended)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QSlider" name="aacQualitySlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>25</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="aacQualitySpin">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="aacBitrateGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Use bitrate</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Bitrate:</string>
        </property>
        <property name="buddy">
         <cstring>aacBitrateCbx</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="aacBitrateCbx">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Sets target bitrate (in kb/s).</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>122</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>aacUseQualityCheck</tabstop>
  <tabstop>aacQualitySlider</tabstop>
  <tabstop>aacQualitySpin</tabstop>
  <tabstop>aacBitrateCbx</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>aacUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>aacBitrateGroup</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aacUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>aacQualitySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aacUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aacUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>aacBitrateCbx</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aacQualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aacQualitySpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aacQualitySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aacQualitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aacUseQualityCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>aacQualitySpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
